### DISCUSSION TITLE
HTML export

### DISCUSSION THREAD

**Discussion by ivandatasci** [Heroor/md-reader#50] [Q&A] [answered]
Is it possible to export as HTML the output rendered by md-reader?

**Comment by Heroor** [comment]
> Is it possible to export as HTML the output rendered by md-reader?

The current version is not supported, but this feature is in the development plan, you can also temporarily through the browser built-in export method, usually `ctrl+s`

**Reply by ivandatasci** [reply]
Just like you, I thought that `ctrl-s` would export the rendered HTML.
It turns out that it doesn't. It just exports the original `.md`.

I am using Google Chrome Stable for linux. Does `ctrl-s` work for you? Would you mind trying again?

Thank you.

Ivan

**Reply by Heroor** [reply, marked as answer]
Sorry, I mistakenly thought it was exportable, I tried and found that it is not possible to export to HTML format.

I will prioritize the development of this feature, please wait for this feature to go live!

**Comment by ivandatasci** [comment]
Thank you for considering this feature for development.
Thank you also for sharing MD-Reader with the world.
